messages/es.json

```
@@ -190,15 +190,15 @@
   "promotions.promotion.scopeSelector.category.verb.any": "es uno de",
   "promotions.promotion.scopeSelector.category.verb.not.any": "no es uno de",
   "promotions.promotion.scopeSelector.category.placeholder": "Seleccione...",
-  "promotions.promotion.scopeSelector.collection.label": "Coleccíon",
+  "promotions.promotion.scopeSelector.collection.label": "Colección",
   "promotions.promotion.scopeSelector.collection.verb.any": "es una de",
   "promotions.promotion.scopeSelector.collection.verb.not.any": "no es una de",
   "promotions.promotion.scopeSelector.collection.placeholder": "Seleccione...",
   "promotions.promotion.scopeSelector.product.label": "Producto",
   "promotions.promotion.scopeSelector.product.verb.any": "es uno de",
   "promotions.promotion.scopeSelector.product.verb.not.any": "no es uno de",
   "promotions.promotion.scopeSelector.product.placeholder": "Seleccione...",
-  "promotions.promotion.scopeSelector.seller.label": "Seller",
+  "promotions.promotion.scopeSelector.seller.label": "Vendedor",
```

Oops i missed this one
```suggestion
  "promotions.promotion.scopeSelector.seller.label": "Seller",
```